libs/domains/services/feature/src/lib/service-terminal/service-terminal.tsx

```diff
@@ -52,23 +59,44 @@ export function ServiceTerminal({
       project: projectId,
       environment: environmentId,
       service: serviceId,
+      pod_name: selectedPod,
+      container_name: selectedContainer,
     },
     onOpen: onOpenHandler,
     onClose: onCloseHandler,
   })
 
   return createPortal(
     <div className="fixed bottom-0 left-0 w-full animate-slidein-up-md-faded">
-      <div className="flex justify-between h-11 px-4 py-2 bg-neutral-650">
-        {/* TODO: add pod_name select */}
-        <span></span>
+      <div className="flex justify-between h-11 px-4 py-2 bg-neutral-650 border-y border-neutral-500">
+        <div className="flex gap-2">
+          {runningStatuses && runningStatuses.pods.length > 0 && (
+            <InputSearch
+              value={selectedPod}
+              onChange={setSelectedPod}
+              data={runningStatuses.pods.map((pod) => pod.name)}
+              placeholder="Search by pod"
+              trimLabel
+            />
+          )}
+          {runningStatuses && selectedPod && (
+            <InputSearch
+              value={selectedContainer}
+              onChange={setSelectedContainer}
+              data={runningStatuses.pods
+                .filter((pod) => selectedPod === pod.name)[0]
```

Comment:
use `find` instead of `filter` and add optional chaining to prevent weird case error

Reply:
https://github.com/Qovery/console/pull/1261/commits/a3154d80dbf4a9dbc6dcf8eeb4a5d31b4c03bf70